migrations/20200323163000-collective-platform-fee.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const OC_FEE_PERCENT = 5;
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Collectives', 'platformFeePercent', {
+      type: Sequelize.INTEGER,
+      defaultValue: OC_FEE_PERCENT,
```

Comment:
If `null` is not an acceptable value for this column, can we add `allowNull`:

```suggestion
      defaultValue: OC_FEE_PERCENT,
      allowNull: false,
```

Reply:
Maybe platformFeePercent doesn't make sense for Organization or User, so it should be nullable and only set for Collectives and Events. We should then make sure to set it when creating new Collectives and Events.

Also, it's weird to have hostFeePercent and platformFeePercent to be of different types (float vs integer). What should we settle on?

Reply:
Looks like integer is better here since it's already a percentage value. Hard to imagine less than 1% precision. My $0.02 ;-)

Reply:
@znarf that's the only inconsistency between `platformFeePercent` and `hostFeePercent`, but reviewing the code I can guarantee nothing will go bad.
Never use float when you can use integer.

Reply:
Like no one ever will want to cut platform fees in half and say 2.5% ? ;-)